src/holder/verifiable_presentation_builder.rs

```diff
@@ -0,0 +1,164 @@
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use serde::{Deserialize, Serialize};
+use ssi_claims::jwt::{VerifiableCredential, VerifiablePresentation};
+use ssi_claims::vc::v2::syntax::VERIFIABLE_PRESENTATION_TYPE;
+use ssi_dids::ssi_json_ld::CREDENTIALS_V1_CONTEXT;
+use ssi_dids::{
+    ssi_json_ld::syntax::{Object, Value},
+    DIDURLBuf,
+};
+
+#[derive(Debug, Clone)]
+pub struct VerifiablePresentationBuilderOptions {
+    pub issuer: DIDURLBuf,
```

Comment:
Is it a requirement of OID4VP that we use DIDs? I know this is not a requirement of VPs.